src/Microsoft.DotNet.ProjectJsonMigration/Rules/IMigrationRule.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+using System.Linq;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
```

Comment:
There's a few places in the cli where we do it like this. 

I don't have a particular preference, but is there a benefit to matching namespaces to folders?